src/test/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValuesTest.java

```diff
@@ -158,6 +158,191 @@ private void verify_empty_Stream_is_returned(String streamFqcn) throws Exception
         assertEquals("count of empty " + streamFqcn, 0L, count);
     }
 
+    @Test
+    public void should_return_empty_duration() throws Exception {
+        //given
+        final String fqcn = "java.time.Duration";
+        final Class<?> durationClass = getClassOrSkipTest(fqcn);
+
+        // when
+        final Object duration = values.returnValueFor(durationClass);
+        final int nano = (Integer) durationClass.getMethod("getNano").invoke(duration);
+        final long seconds = (Long) durationClass.getMethod("getSeconds").invoke(duration);
+
+        // then
+        assertEquals("nano of empty " + fqcn, 0, nano);
+        assertEquals("seconds of empty " + fqcn, 0L, seconds);
+    }
+
+    @Test
+    public void should_return_empty_period() throws Exception {
+        //given
+        final String fqcn = "java.time.Period";
+        final Class<?> periodClass = getClassOrSkipTest(fqcn);
+
+        // when
+        final Object period = values.returnValueFor(periodClass);
+        final int years = (Integer) periodClass.getMethod("getYears").invoke(period);
+        final int months = (Integer) periodClass.getMethod("getMonths").invoke(period);
+        final int days = (Integer) periodClass.getMethod("getDays").invoke(period);
+
+        // then
+        assertEquals("years of empty " + fqcn, 0, years);
+        assertEquals("months of empty " + fqcn, 0, months);
+        assertEquals("days of empty " + fqcn, 0, days);
+    }
+
+    @Test
+    public void should_return_empty_instant() throws Exception {
+        //given
+        final String fqcn = "java.time.Instant";
+        final Class<?> instantClass = getClassOrSkipTest(fqcn);
+
+        // when
+        final Object instant = values.returnValueFor(instantClass);
+        final int nano = (Integer) instantClass.getMethod("getNano").invoke(instant);
+        final long epochSecond = (Long) instantClass.getMethod("getEpochSecond").invoke(instant);
+
+        // then
+        assertEquals("nano of empty " + fqcn, 0, nano);
+        assertEquals("epoch second of empty " + fqcn, 0L, epochSecond);
+    }
+
+    @Test
+    public void should_return_empty_local_date() throws Exception {
```

Comment:
Seems like this one is failing on JDK 8: https://travis-ci.org/github/mockito/mockito/jobs/666534233#L914

Reply:
yes, EPOCH static was from jdk11 apparently. anyway, after giving a second thought, I decided to leave only period and duration, because it's not obvious that empty date is epoch.